node/user.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2020 - for information on the respective copyright owner
+// see the NOTICE file and/or the repository at
+// https://github.com/direct-state-transfer/dst-go
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package node
+
+import (
+	"github.com/pkg/errors"
+	"perun.network/go-perun/wallet"
+
+	"github.com/direct-state-transfer/dst-go"
+)
+
+// NewUnlockedUser initializes a user and unlocks all the accounts.
+// (those corresponding to on-chain address, off-chain address and all participant addresses.
+func NewUnlockedUser(wb dst.WalletBackend, cfg UserConfig) (dst.User, error) {
+	if wb == nil {
+		return dst.User{}, errors.New("wallet backend should not be nil")
+	}
+	var err error
```

Comment:
Use named returns.

Reply:
I find named returns useful only when it adds more meaning to the function signature. Do you feel, having named returns adds more meaning ?

I also find this is one of the common good practices from the [google code review comments](https://gist.github.com/adamveld12/c0d9f0d5f0e1fba1e551#named-result-parameters).

Reply:
I normally just do it if it makes the function shorter :laughing:

Reply:
Yeah, I was also doing the same. :smiley:

But I found it appealing that, doing it this way makes it look better in the docs, so adopted it. :wink: